tuftool/CHANGELOG.md

```diff
@@ -4,10 +4,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [0.4.1] - 2020-07-20
+### Added
+- Added logging.
+- Added downloading of specific targets.
+- Allow control of link/copy behavior for existing paths.
+
 ### Changed
-- Bumped `tough` to 0.7.0
-- Bumped `tough-ssm` to 0.2.0
+- Bumped `tough` to 0.8.0.
```

Comment:
(Just to pre-answer the question, because tuftool doesn't have exported APIs that rely on the version of tough, this does not require a bump to 0.5.0)